### ISSUE TITLE
missing 'hidden tags' icon

### ISSUE THREAD

**Issue by jkhsjdhjs** [munlik/refind-theme-regular#8]
The following icon is missing in the theme: https://sourceforge.net/p/refind/code/ci/master/tree/icons/func_hidden.png

**Comment by voviks21** [comment]
yes, looks not good. Did anyone found any replacement for missing icon? I would try so..

**Comment by d4k0** [comment]
I switched to this theme a few days ago and I had the same problem. That's why I created icons for `func_hidden.png` and `tool_fwupdate.png` (this seems to be used by Ubuntu) using freely available vector images. The images have been optimized for minimal size by using FileOptimizer. The icons use the same color as the icons of this theme. Have fun with it!

![func_hidden](https://user-images.githubusercontent.com/10982825/42830738-94743322-89ec-11e8-8138-2b57a2118244.png)
![tool_fwupdate](https://user-images.githubusercontent.com/10982825/42830681-73002462-89ec-11e8-9883-6a8cb221ffdf.png)
[refind_theme_regular_icons.zip](https://github.com/munlik/refind-theme-regular/files/2202444/refind_theme_regular_icons.zip)

**Comment by jkhsjdhjs** [comment]
Can't even describe how much I appreciate these, thanks a lot for sharing! Finally my bootloader has its beauty back!

**Comment by bobafetthotmail** [comment]
@d4k0 They look great! Do you also have a SVG file for them too? I want to add them to my fork here https://github.com/bobafetthotmail/refind-theme-regular (or you can send a PR yourself)

**Comment by d4k0** [comment]
Glad that all of you like them :).

@bobafetthotmail I attached SVG versions of the icons, I hope they are alright. I'm a bit busy at the moment, so you can add them to your fork.

[refind_theme_regular_icons_svg.zip](https://github.com/munlik/refind-theme-regular/files/2211275/refind_theme_regular_icons_svg.zip)

**Comment by bobafetthotmail** [comment]
Thanks a lot, I'll credit you in the commit. :)

**Comment by bobafetthotmail** [comment]
ok done.

**Comment by d4k0** [comment]
Well, I measured the original icons itself (without the transparent border), exported the new icons at these sizes (the smallest ones where around 25x25 I think) and pasted them into the final images with transparent background (48x48 is the smallest size).

Do the icons that are exported by the script work without a problem as they seem to have much less border/are smaller? I could also try to edit the SVG files so that they fit the other icons.

**Comment by d4k0** [comment]
I created new versions based on `func_reset.svg`. They now also have the transparent border like the other icons. I tried to match the size of of the other icons.

[refind_theme_regular_icons_svg.zip](https://github.com/munlik/refind-theme-regular/files/2233839/refind_theme_regular_icons_svg.zip)

**Comment by bobafetthotmail** [comment]
Thanks, integrated again. 

The svg files actually need to have space all around like that, I used the wrong image for tests and I didn't notice.

**Comment by d4k0** [comment]
Glad I could help :).

**Comment by Cactus64k** [comment]
> I switched to this theme a few days ago and I had the same problem. That's why I created icons for `func_hidden.png` and `tool_fwupdate.png` (this seems to be used by Ubuntu) using freely available vector images. The images have been optimized for minimal size by using FileOptimizer. The icons use the same color as the icons of this theme. Have fun with it!
> 
> ![func_hidden](https://user-images.githubusercontent.com/10982825/42830738-94743322-89ec-11e8-8138-2b57a2118244.png)
> ![tool_fwupdate](https://user-images.githubusercontent.com/10982825/42830681-73002462-89ec-11e8-9883-6a8cb221ffdf.png)
> [refind_theme_regular_icons.zip](https://github.com/munlik/refind-theme-regular/files/2202444/refind_theme_regular_icons.zip)

Can you create fork with this fix?
